packages/app-bridge/src/types/Color.ts

```diff
@@ -94,8 +94,17 @@ export type ColorApiPatch = Partial<{
 
 export type Color = {
     id: number;
-    name: Nullable<string>;
     sort: number;
+    color: ColorAsset;
+};
+
+// This type should be evolved to Asset in the long run
+export type ColorAsset = {
+    title: Nullable<string>;
+    revision: ColorRevision;
+};
```

Comment:
🤔 I reckon not all properties of `Asset` are relevant for the different asset types (at least at the moment). Doesn't that point towards Asset being an interface rather than a type with different revision properties only? Or am I missing something? Are we moving towards having a stable Asset type where the specific type properties are then defined in these different Revision sub-types?

Reply:
As far I know, the asset type shouldn't have anything special which only is used for a special type of `revision`, as you only can only decide on the `payload` of the `revision` what type of an `asset` it it. Everything on the `asset` could/should be used by all assets.
Probably something we should keep in mind as it evolves in the future.